db/migrate/20231117134407_add_fields_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddFieldsToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :birth, :date
```

Comment:
if you are validating the presence of `birth` in the user model, then this collumn should not contain null values.
Change this migration so it looks like:
```ruby
add_column :users, :birth, :date, null: false
```